src/pazi_result.rs

```diff
@@ -16,19 +16,21 @@ macro_rules! EXIT_CODE {
     (SUCCESS) => { 90 };
     (SUCCESS_DIR) => { 91 };
     (ERROR) => { 92 };
+    (SUCCESS_DID_NOTHING) => { 93 };
```

Comment:
I don't like `Success` in the name since that implies returning zero to me.

Maybe `ERROR_NO_OUTPUT` or `EXPECTED_ERROR` or such?

Reply:
This variant also needs to be handled in pazi init for all shells, e.g. in

https://github.com/euank/pazi/blob/07fd9e7fcb2c8d33d790340f28b6d67323a7e262/src/shells/zsh.rs#L23-L34

Perhaps there should be a dummy rust-switch in that function on `PaziResult` to force compiler errors if someone forgets to change it? Or maybe it can just remain a sorta subtle footgun